src/ukbb_common/utils/saige_pipeline.py

```diff
@@ -8,7 +8,7 @@
 
 
 MKL_OFF = 'export MKL_NUM_THREADS=1; export MKL_DYNAMIC=false; export OMP_NUM_THREADS=1; export OMP_DYNAMIC=false; '
-SCRIPT_DIR = '/ukb_common/saige'
+SCRIPT_DIR = '/ukbb_common/saige'
```

Comment:
@konradjk I think this line suggests the docker image already has `ukb_common` in it. Happy to keep `ukb_common` for now or please update the docker too

Reply:
Probably keep for now and I can update when I update the dockers next

Reply:
I think since I'm the next person to use the dockers, fine to update and I will update when it merges/next time I use it